Makefile

```diff
@@ -87,7 +85,7 @@ packaging/arch/PKGBUILD: packaging/arch/PKGBUILD.in
 packaging/debian/changelog: packaging/debian/changelog.in
 	sed 's/VERSION/$(VERSION)/' $< > $@
 
-$(WEBPACK_TEST): $(LIB_TEST) $(shell find src/ -type f) package.json webpack.config.js
+$(WEBPACK_TEST): pkg/lib $(shell find src/ -type f) package.json webpack.config.js
```

Comment:
I deliberately used file dependencies here instead of directory; I had the impression/experience that make dependencies for directories are rather unreliable.  Also, there is no actual rule to produce `pkg/lib`, so it feels like this would just fail if pkg/lib is not present?

Was there something particularly wrong with `LIB_TEST=pkg/lib/cockpit-po-plugin.js`? You introduce `COCKPIT_REPO_STAMP` below with the same value, ISTM that the definition should move to the top and be used here as well.

Reply:
Ya.  I can put it back.  It's possible to imagine a situation where the `Makefile` has been modified to check out a different version, but `make` considers `pkg/lib` newer because some random editor temp file got left there...

It just feels a bit evil to randomly elect a file for this purpose.  I guess now that I use 267, `cockpit.js` is a good choice.  Although, ironically: it's the file that we *don't* use :)